example.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 #
 # Copyright (c) 2018 Sam Wenke (samwenke@gmail.com)
+# Copyright (c) 2019 Ingvar Lond (ingvar.lond@gmail.com)
```

Comment:
It's fine by me to add a copyright note, but it's less fine to sneak it in a "bugfix" commit. Commit messages should ideally reflect all the changes made, even if they're "bureaucrat" ones.

However, at this point I'd like to defer to @wenkesj as to whether he's fine with this addition, as I don't feel comfortable merging this in.

Reply:
Sorry about that. Doesn't have to be there if you don't like it.